src/lightning_app/core/queues.py

```diff
@@ -304,6 +331,10 @@ def is_running(self) -> bool:
         except redis.exceptions.ConnectionError:
             return False
 
+    def get_all(self, timeout: float):
+        time.sleep(timeout)
+        return self.redis.lrange(self.name, 0, -1)
```

Comment:
we should probably put some limit on this number - if the size of the response is very huge

Reply:
I don't think so. get_all should return them all. Otherwise it would be `batch_get` method.